pep257.py

```diff
@@ -148,6 +150,10 @@ class Method(Function):
 
     @property
     def is_public(self):
+        # Check if we are a setter/deleter method, and mark as private if so.
+        for decorator in self.decorators:
+            if decorator.name.startswith(self.name):
```

Comment:
Also check that after `self.name` there is a period, otherwise this will be considered private:

````
class Foo(object):
    @foobar
    def foo():
        pass
````